cli.js

```diff
@@ -4,6 +4,18 @@ import fs from 'node:fs'
 import sortPackageJson from './index.js'
 import Reporter from './reporter.js'
 
+/** @param {import("node:stream").Stream} stream */
+async function streamToString(stream) {
+  // see https://stackoverflow.com/a/63361543/6051261
+  const chunks = []
+
+  for await (const chunk of stream) {
+    chunks.push(Buffer.from(chunk))
+  }
+
+  return Buffer.concat(chunks).toString('utf-8')
+}
```

Comment:
We can use https://www.npmjs.com/package/get-stdin

Reply:
:+1: implemented in [keithamus/sort-package-json@`432780d` (#296)](https://github.com/keithamus/sort-package-json/pull/296/commits/432780de473169be5d0bde4c8b2e7236e161fbaf)